Healthy/Classes/Modules/MainTabBarController/UITabBarController+Style.swift

```diff
@@ -0,0 +1,23 @@
+import UIKit
+
+extension UITabBarController {
```

Comment:
```suggestion
extension MainTabBarController {
```
Also, shuld be moved to the same file of MainTabBarController